js/viz/vector_map/control_bar.js

```diff
@@ -215,6 +229,14 @@ ControlBar.prototype = {
         return this._isActive ? this._layoutOptions : null;
     },
 
+    getChildrenPanControl: function() {
```

Comment:
This code uses in tests only. And it will be in the production code. We should to wrap this code in ///#DEBUG ///#ENDDEBUG or rewrite the tests